test/src/sender/main_sender.cpp

```diff
@@ -22,10 +22,14 @@
 #include <cadmium/logger/tuple_to_ostream.hpp>
 #include <cadmium/logger/common_loggers.hpp>
 
-#include "../../../lib/vendor/NDTime.hpp"
+#include "../../../lib/DESTimes/include/NDTime.hpp"
 #include "../../../lib/vendor/iestream.hpp"
 #include "../../../include/data_structures/message.hpp"
 #include "../../../include/atomics/sender_cadmium.hpp"
+#include "../../../include/output_modification/convert_output.hpp"
+
+#define SIMULATOR_OUTPUT_PATH "./test/data/sender_test_output.txt"
+#define MODIFIED_OUTPUT_FILE "./test/data/modified/converted_sender_output.txt"
```

Comment:
name is not self-explanatory. see previous comments and update accordingly.

Reply:
you can create one more #define statement for line 131 and 148 and replace file path with #define statement

string input_data_control = "test/data/sender_input_test_control_In.txt"; 
string input_data_ack = "test/data/sender_input_test_ack_In.txt";